packages/vscode-extension/package.nls.json

```diff
@@ -1,5 +1,13 @@
 {
     "teamstoolkit.accountTree.azureAccountTooltip": "AZURE ACCOUNT  \nThe Teams Toolkit may require an Azure subscription to deploy the Azure resources for your project.",
+    "teamstoolkit.accountTree.copilotEnroll": "Enroll in Early Access Program",
+    "teamstoolkit.accountTree.copilotMessage": "Microsoft 365 account admin haven't enabled Copilot access for this account. Contact your Teams administrator to resolve this issue by enrolling in Microsoft 365 Copilot Early Access program.",
+    "teamstoolkit.accountTree.copilotPass": "Copilot Access Enabled",
+    "teamstoolkit.accountTree.copilotPassTooltip": "You have Copilot access already.",
+    "teamstoolkit.accountTree.copilotStatusUnknown": "Copilot Access Check Failed",
+    "teamstoolkit.accountTree.copilotStatusUnknownTooltip": "The Copilot access of this account is unknown due to network or service issue. Please refresh again later to check it.",
+    "teamstoolkit.accountTree.copilotWarning": "Copilot Access Disabled",
+    "teamstoolkit.accountTree.copilotWarningTooltip": "Microsoft 365 account admin haven't enabled Copilot access for this account.\nContact your Teams administrator to resolve this issue by enrolling in Microsoft 365 Copilot Early Access program.\nVisit: https://aka.ms/PluginsEarlyAccess",
```

Comment:
Better extract the link from the string for better localization practice.

Reply:
This string is referenced in `package.json` as command title. https://github.com/OfficeDev/TeamsFx/blob/qidon/copilotcheck/packages/vscode-extension/package.json#L748

Does that support extraction as well?

Reply:
OK, then no need to extract~